examples/tutorials/starting/overview.py

```diff
@@ -75,6 +75,10 @@
 # Check setup
 # -----------
 from gammapy.utils.check import check_tutorials_setup
+from gammapy.maps import Map
+from gammapy.data import EventList
+from gammapy.utils.regions import SphericalCircleSkyRegion
+from gammapy.catalog import SourceCatalog3FHL
```

Comment:
In this case too, I would not change the import order.

Reply:
it's done, please check if everything is ok now